packages/expo-updates/ios/EXUpdates/AppLoader/EXUpdatesFileDownloader.m

```diff
@@ -143,8 +144,14 @@ - (void)parseManifestResponse:(NSHTTPURLResponse *)httpResponse
                                    successBlock:successBlock
                                      errorBlock:errorBlock];
   } else {
+    NSDictionary *responseHeaders = [httpResponse allHeaderFields];
+    EXUpdatesManifestHeaders *manifestHeaders = [[EXUpdatesManifestHeaders alloc] initWithProtocolVersion:responseHeaders[@"expo-protocol-version"]
```

Comment:
I feel better about using valueForHTTPHeaderField instead, which handles case-insensitivity compared to allHeaderFields trying to canonicalize casing (it doesn't just lowercase)

Reply:
Unfortunately it's only available on iOS 13+: https://developer.apple.com/documentation/foundation/nshttpurlresponse/3240613-valueforhttpheaderfield?language=objc

Reply:
We'll probably drop iOS 12 sometime after the iOS 16 release or perhaps earlier (Google Maps is dropping it) so I'd leave a comment here saying we can use the method after dropping iOS 12.